testdata/jsonschema/basic_object/GenerateAST/ir.json

```diff
@@ -136,6 +136,63 @@
               },
               "Required": false
             },
+            {
+              "Name": "objectAdditionalPropertiesAny",
+              "Type": {
+                "Kind": "scalar",
+                "Nullable": false,
+                "Scalar": {
+                  "ScalarKind": "any"
```

Comment:
This doesn't seem like it's doing the expected thing right? It should be a map of string -> object, right?

Example:
```
{
	"objectAdditionalPropertiesAny": {
    "hello": "hello"
    }
}
```
fails with
```
Invalid type. Expected Object but got String. Path 'objectAdditionalPropertiesAny.hello', line 3, position 20.
```
when I try to validate it. The AST doesn't seem to reflect that behavior

Reply:
You're definitely correct! :facepalm: 

I'll close this PR :+1: